src/Microsoft.AspNetCore.JsonPatch/project.json

```diff
@@ -21,22 +21,17 @@
     ]
   },
   "dependencies": {
+    "Microsoft.Extensions.ClosedGenericMatcher.Sources": {
+      "type": "build",
+      "version": "1.1.0-*"
+    },
     "Newtonsoft.Json": "9.0.1"
   },
   "frameworks": {
-    "netstandard1.1": {
+    "net451": {},
+    "netstandard1.3": {
```

Comment:
I thought about the version, if JsonPatch was used along with MVC then it should not be a problem as MVC requires netstandard1.6, and if its was used outside of MVC then if we consider HttpAbstractions, it requires netstandard1.3. Considering these this change should be fine?